grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java

```diff
@@ -878,10 +878,10 @@ public String authority() {
       MethodDescriptor<ReqT, RespT> methodDescriptor, CallOptions callOptions) {
```

Comment:
nit: why does this line have 2 extra spaces at the start?

Reply:
I guess `googleJavaFormat` makes a 4-space indentation for a multiline method signature.